### ISSUE TITLE
credentials via sts get-session-token possible?

### ISSUE THREAD

**Issue by hflamboauto1** [twistlock/cloud-discovery#7]
This is most basically a question to the fact if is possible to use  credentials retrieved from STS

```
aws sts get-session-token
```

**Comment by liron-l** [comment]
@hflamboauto1 yes, this suppose to work (I assume the token lifespan is sufficient for a single scan). Please let me know if you get any errors.
